cmd/insprd/operators/kafka/nodes/converter.go

```diff
@@ -98,6 +98,7 @@ func (no *NodeOperator) withBoundary(app *meta.App) k8s.ContainerOption {
 			parent, chName, _ := metautils.RemoveLastPartInScope(resolved)
 			ch, _ := no.memory.Channels().Get(parent, chName)
 			ct, _ := no.memory.Types().Get(parent, ch.Spec.Type)
+			env[boundary+"_RESOLVED_SCOPE"] = resolved
```

Comment:
What does this do?

Reply:
This stores the scope/path to the Channel (something like "app1.app2.chan1"). This is necessary for the LB to know which sidecar the request will be sent to - the LB receives the (unresolved) channel where the message is going to be sent through, gets this channel in-memory structure to know it's broker and then proceeds to get the specific broker ports to send the request.